framework/doc/content/source/fviks/FVOneVarDiffusionInterface.md

```diff
@@ -0,0 +1,19 @@
+# FVOneVarDiffusionInterface
+
+!syntax description /FVInterfaceKernels/FVOneVarDiffusionInterface
+
+This class is only meant to be used with a single variable, e.g. the `variable1`
+parameter should be provided but not `variable2`. `coeff1` should correspond to
+the diffusion or conductivity coefficient on the `subdomain1` side of the
+`boundary` while `coeff2` corresponds to `subdomain2`. The gradient of
+`variable1` is based on Green-Gauss computations of the gradient at the
+adjoining element centroids followed by linear interpolation to the face plus a
+non-orthogonal correction. The diffusion/conductivity coefficient at the
+interface is computed using linear interpolation of the `coeff1` and `coeff2`
+values. This discretization scheme is O(h) accurate.
```

Comment:
why is it only first order accurate (in the test)?
Isn't our regular diffusion discretization second order on cartesian meshes?

Reply:
I believe it's only first order because of the way we separate the interpolation of the diffusion coefficient and the gradient. You raise a good point. I'll see about making a test without an `FVInterfaceKernel` with a similar step change in the diffusion coefficient value; if my hypothesis is correct, that should also show only first order, independent of use of `FVInterfaceKernel`.

Reply:
yea I added a test without the interface kernel and convergence is still first order